lua/chainsaw/log-statements-data.lua

```diff
@@ -16,15 +17,17 @@ return {
 	},
 	objectLog = {
 		nvim_lua = 'vim.notify("%s %s: " .. vim.inspect(%s))',
-		typescript = 'console.log("%s %s:", %s)',
-		javascript = 'console.log("%s %s:", JSON.stringify(%s))',
+		typescript = 'console.log("%s %s:", JSON.stringify(%s))',
```

Comment:
To my knowledge, typescript automatically stringifies for `console.log`, so it is not necessary to add it here

Reply:
No, When `TS` is converted to `JS`, the content of `console.log` will not be enchanted.

When the browser prints, you will see a pure object (not a string), which needs to be looked up step by step (of course, some users will prefer this way)
<img width="298" alt="image" src="https://github.com/chrisgrieser/nvim-chainsaw/assets/63690944/5c405443-77f0-4a34-84a4-811e22f2b717">

If it is in a node.js scene, it will be displayed on the console in an omitted manner (you can't see the full picture of the object directly)

<img width="731" alt="image" src="https://github.com/chrisgrieser/nvim-chainsaw/assets/63690944/cae4048a-e846-4fbd-ab1f-626c528f408f">

Reply:
oh okay, good point